### PR TITLE
Add staticroute e2e test

### PR DESCRIPTION
**Pull request by TaoZou1**
Add staticroute e2e test
Update framework timeout value

### CODE REVIEW

File: test/e2e/framework.go [hunk 1]
```diff
@@ -29,9 +29,10 @@ import (
 )
 
 const (
-	defaultTimeout         = 100 * time.Second
+	defaultTimeout         = 300 * time.Second
```
Comment (dantingl) [inline test/e2e/framework.go:32]: Does it mean static route need more time to realize?

Comment (TaoZou1) [reply test/e2e/framework.go:32]: 100s is not enough for some case. I met the case while did e2e test.

**Comment by zhengxiexie** [comment]
pls commit a PR for https://gitlab.eng.vmware.com/nsx-ujo/nsx-operator-ci/-/blob/main/main.py#L366

**Pull request closed without merging**